protocol/storage/fault_slasher_test.go

```diff
@@ -50,18 +54,23 @@ func TestFaultSlasher_Slash(t *testing.T) {
 	tf.UnitTest(t)
 
 	ctx := context.Background()
-	signer, _ := types.NewMockSignersAndKeyInfo(1)
+	signer, _ := types.NewMockSignersAndKeyInfo(3)
 
 	t.Run("When there are no miners, does not error", func(t *testing.T) {
 
 		height := types.NewBlockHeight(1)
 		data, err := cbor.DumpObject(&map[string]uint64{})
 		require.NoError(t, err)
 		queryer := makeQueryer([][]byte{data})
-		minerOwnerAddr := signer.Addresses[0]
 
 		ob := outbox{}
-		fm := NewFaultSlasher(&slasherPlumbing{false, false, queryer}, &ob, minerOwnerAddr, DefaultFaultSlasherGasPrice, DefaultFaultSlasherGasLimit)
+		sp := slasherPlumbing{
+			Queryer:    queryer,
+			minerAddr:  signer.Addresses[0],
+			workerAddr: signer.Addresses[1],
+		}
```

Comment:
Factor out this oft-duplicated block?

Reply:
I just tried to factor this and there are just enough small differences in various that it doesn't really help much.